prboom2/src/hu_stuff.c

```diff
@@ -65,9 +65,7 @@
 
 static player_t*  plr;
 
-static dboolean    always_off = false;
```

Comment:
Were these supposed to do something?

Reply:
Looks like this was used somehow in the net chat components. It can go.

Reply:
Yep, this was used to initialize the input buffers for netgame players.